core/tee/tee_svc.c

```diff
@@ -588,22 +588,28 @@ static TEE_Result utee_param_to_param(struct user_ta_ctx *utc,
 				      struct utee_params *up)
 {
 	size_t n = 0;
-	uint32_t types = up->types;
+	uint64_t types = 0;
+
+	GET_USER_SCALAR(types, &up->types);
```

Comment:
How about copying `up` into a bounce buffer instead?

Reply:
I updated this function to use `up_bbuf` allocated from the bounce buffer pool.